### PR TITLE
Update forms

### LINKED ISSUE #167: Add an organization edit button
It would be nice to have an edit button on the organization page, which is similar to the edit button on the organizations list page.

From a workflow perspective, the difference between the two is that clicking on the org edit button, should take you to the form with the organization pre-selected. 

Here on some docs on pre-filling a form https://support.airtable.com/hc/en-us/articles/234982508-Prefilling-a-form

### PR DESCRIPTION
**Pull request by jasonLaster**
This updates the organization/capital add/edit forms so that on the list pages you add and the item pages you edit.

Fixes #167 

The goal is to simplify the flow:

1. when adding you're not prompted to find an organization
2. when editing, you're not prompted to find the organization you're editing

If we want to we can prefill some of the other fields, but i don't think that's necessary and could make it harder to show what each person changed

I dropped the add organization section at the bottom because the Add button is super prominent

### CODE REVIEW

File: src/templates/organization.js [hunk 1]
```diff
@@ -73,7 +79,8 @@ function Tags({ org }) {
 }
 
 export default function OrganizationTemplate({ data }) {
-  const siteTitle = data.site.siteMetadata.title
+  const { siteTitle } = data.site.siteMetadata
```
Comment (bloudermilk) [inline src/templates/organization.js]: Is this a regression? Would expect it to be `const { title }` to keep the same value.

File: src/templates/organization.js [hunk 2]
```diff
@@ -21,6 +21,12 @@ function getLogoImage({ logo, photos, categories }) {
     categories.find(c => c.cover) || categories.find(c => c?.parent?.cover)
   return logo || photos[0] || cat?.cover || cat?.parent.cover
 }
+function getEditUrl({ data, org }) {
+  const { capitalEditFormUrl, organizationEditFormUrl } = data.site.siteMetadata
+  const url =
+    data.type === "organization" ? organizationEditFormUrl : capitalEditFormUrl
```
Comment (bloudermilk) [inline src/templates/organization.js]: What is `data.type` here? I think we should be checking if `role` includes `"Capital"`

File: src/templates/organization.js (hunk 1 above)
Comment (jasonLaster) [reply src/templates/organization.js]: yep

File: src/templates/organization.js (hunk 2 above)
Comment (jasonLaster) [reply src/templates/organization.js]: good call

**Comment by bloudermilk** [comment]
@jasonLaster looks great when running locally. If we can figure out the build problem I think this is good to merge!

**Pull request merged**